_templates/_page_openshift.html.erb

```diff
@@ -270,21 +286,29 @@
   <!-- adjust page css based on breadcrumb and/or resize -->
   <script type="text/javascript">
     window.addEventListener('DOMContentLoaded', () => {
-      if (window.innerWidth >= 1008) {
+      if ($(window).width() >= 1008) {
```

Comment:
The switch here away from `innerWidth` addresses a Safari issue where `innerWidth` isn't actually accurate. So there was an unknown bug in Safari where resizing the browser caused an issue where the sidebar was shown and hidden at the same time

Reply:
https://user-images.githubusercontent.com/22222756/154157516-941784d2-31de-47f0-9dfd-06d6955ac9af.mov